invokeai/backend/model_manager/load/model_loaders/lora.py

```diff
@@ -68,7 +68,13 @@ def _load_model(
             model = lora_model_from_sd_state_dict(state_dict=state_dict)
         elif self._model_base == BaseModelType.Flux:
             if config.format == ModelFormat.Diffusers:
-                model = lora_model_from_flux_diffusers_state_dict(state_dict=state_dict)
+                # HACK(ryand): We assume alpha=8 for diffusers PEFT format models. These models are typically
+                # distributed as a single file without the associated metadata containing the alpha value. We chose
+                # alpha=8, because this is the default value in the PEFT library:
```

Comment:
lol it's alpha=dim

Reply:
Agreed that's probably a more common choice. I'll fix this up.

Are you going based on https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/train_dreambooth_lora_flux.py#L1194? Or something else?

Reply:
yeah we've got some stuff WIP for making alpha an encoded value in the diffusers (peft) LoRAs but it's not really widely available yet, and diffusers' assumption is alpha=dim

Reply:
Addressed in [e88d3cf](https://github.com/invoke-ai/InvokeAI/pull/6847/commits/e88d3cf2f7459c8ce1ae56e1ee72f5c2a9976fb6)